draft-ietf-quic-transport.md

```diff
@@ -2225,14 +2226,11 @@ connection state.  This results in new packets on the connection being handled
 generically.  For instance, an endpoint MAY send a stateless reset in response
 to any further incoming packets.
 
-The draining and closing periods do not apply when a stateless reset
-({{stateless-reset}}) is sent.
-
```

Comment:
What do you hope to achieve by removing this text?  It's still accurate, right?

Reply:
Closing and draining applies to connection. Stateless reset is sent when there is no connection, so I have no idea how this sentence would apply.

Reply:
That is the whole point of the text.  You could expand it to say that this is because there is no connection, I suppose.

Reply:
As it stands, it sounds like instead of going through draining and closing period, I can choose a shortcut and just send a stateless reset to kill a connection. I have no idea how one would even start the closing period if there was no connection in the first place.